components/license-control-service/pkg/server/server.go

```diff
@@ -155,8 +156,9 @@ func (s *LicenseControlServer) Update(ctx context.Context, req *lc.UpdateRequest
 	)
 
 	if !req.Force && ptypes.TimestampNow().GetSeconds() > licensedPeriod.end.GetSeconds() {
-		msg := fmt.Sprintf("Rejecting license ID %v, expired at %v; set force=true to override",
-			license.Id, licensedPeriod.end)
+		msg := fmt.Sprintf("Rejecting license ID %v, expired at %s; set force=true to override",
+			license.Id,
+			time.Unix(licensedPeriod.end.GetSeconds(), 0).UTC().Format(time.RFC3339))
```

Comment:
Found this pattern elsewhere in the codebase; seems like should be able to shorten it I would think, but this works. 🤷‍♂